tests/unit/dom/select-files-test.js

```diff
@@ -0,0 +1,53 @@
+import { module, test } from 'qunit';
+import { triggerEvent, setupContext, teardownContext } from '@ember/test-helpers';
+import { buildInstrumentedElement } from '../../helpers/events';
+import hasEmberVersion from 'ember-test-helpers/has-ember-version';
+
+module('DOM Helper: selectFiles', function(hooks) {
+  if (!hasEmberVersion(2, 4)) {
+    return;
+  }
+
+  let context, element;
+
+  const textFile = new Blob(['Hello World'], { type: 'text/plain' });
+  textFile.name = 'text-file.txt';
+
+  const imageFile = new Blob([], { type: 'text/png' });
+  imageFile.name = 'image-file.png';
+
+  hooks.beforeEach(function() {
+    // used to simulate how `setupRenderingTest` (and soon `setupApplicationTest`)
```

Comment:
I think this is ok, but it might be better to _actually_ use `setupRenderingTest` (may end up being less brittle over time), what do you think?

Reply:
I just copied an example from one of the existing helper tests. I'm not sure I have enough knowledge to get this PR in a good enough state to go in. I just made it to demonstrate what I need.

Reply:
Ya, no worries about that. There really isn’t an issue with it the way it is. Just made me reconsider RE: the other tests like this